pkg/v1/tkg/clusterclient/clusterclient.go

```diff
@@ -68,8 +68,8 @@ import (
 	configv1alpha1 "github.com/vmware-tanzu/tanzu-framework/apis/config/v1alpha1"
 	runv1alpha1 "github.com/vmware-tanzu/tanzu-framework/apis/run/v1alpha1"
 	runv1alpha3 "github.com/vmware-tanzu/tanzu-framework/apis/run/v1alpha3"
+	capdiscovery "github.com/vmware-tanzu/tanzu-framework/capabilities/discovery"
```

Comment:
Optional. I like `capdisc` better. Seems sleeker an alias and doesn't lose meaning.